New Basemaps: Utility Network Category (#610)
Esri/arcgis-runtime-samples-java: utility_network/trace-a-utility-network/README.md

## How it works

> 1. Create an `ArcGISMap` and add it to a `MapView`.

Reviewer: since we're updating this anyway:

```suggestion
1. Create an `ArcGISMap` and set it on a `MapView`.
```
